spanner/spanner_samples.rb

```diff
@@ -40,6 +40,38 @@ def create_instance project_id:, instance_id:
   # [END spanner_create_instance]
 end
 
+def create_instance_with_processing_units project_id:, instance_id:
+  # [START spanner_create_instance_with_processing_units]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+
+  require "google/cloud/spanner"
+
+  spanner  = Google::Cloud::Spanner.new project: project_id
+  instance = spanner.instance instance_id
```

Comment:
delete?

Reply:
This `instance` variable is never used. I think this line should be deleted.